zilliqa/src/exec.rs

```diff
@@ -1557,3 +1561,15 @@ fn scilla_call(
         state.take().expect("missing state"),
     ))
 }
+
+/// Blessed transactions bypass minimum gas price rules. These transactions have value to the network even at a lower
+/// gas price, so we accept them anyway.
+const BLESSED_TRANSACTIONS: [Hash; 1] = [
+    // Hash of the deployment transaction for the deterministic deployment proxy from
+    // https://github.com/Arachnid/deterministic-deployment-proxy. It is valuable to accept this transaction despite
+    // the low gas price, because it means the contract is deployed at the same address as other EVM-compatible chains.
+    // This means that contracts deployed using this proxy will be deployed to the same address as on other chains.
+    Hash(hex!(
+        "eddf9e61fb9d8f5111840daef55e5fde0041f5702856532cdbb5a02998033d26"
```

Comment:
https://github.com/Zilliqa/dev-portal/issues/21 is the issue to document this (though probably I now need to start writing a page for all this not-API-but-properties-of-this-version-of-Zilliqa code. Meh.)

Reply:
Would also be good to know what drives putting this txn in the list? Agree that it's useful, but would want to see passive acceptance on slack (at least) before we start adding too much to this array ...

Reply:
I tried to write the motivation in the comment above

> It is valuable to accept this transaction despite the low gas price, because it means the contract is deployed at the same address as other EVM-compatible chains. This means that contracts deployed using this proxy will be deployed to the same address as on other chains.

Does that make sense or need adding to?

Reply:
How about I file an issue, write something in docs/ and then post to slack for visibility? I neither want this to come as a surprise to future developers, nor do I want people to start adding to this list "because it's useful ..". But let's not hold this PR up for that?

Reply:
Okay with me, thank you :)